tools/build_langserver/langserver/hover_test.go

```diff
@@ -0,0 +1,26 @@
+package langserver
+
+import (
+	"context"
+	"core"
+	"path"
+	"testing"
+	"tools/build_langserver/lsp"
+
+)
+
+func TestGetHoverContent(t *testing.T) {
+	core.FindRepoRoot()
+	ctx := context.Background()
+	filepath := path.Join(core.RepoRoot, "tools/build_langserver/langserver/BUILD")
```

Comment:
I don't think this works - you can't read your own build file in a test. You should add a specific test file with what you need in it.